dotnet-authserver/tests/TeacherIdentity.AuthServer.TestCommon/TestData.CreateUser.cs

```diff
@@ -15,7 +14,7 @@ public partial class TestData
             TrnLookupStatus? trnLookupStatus = null,
             string? firstName = null,
             string[]? staffRoles = null,
-            bool? hasMobileNumber = true) =>
+            bool? hasMobileNumber = null) =>
```

Comment:
I changed it to `null` so that `Staff` users wouldn't get assigned a mobile by default (since they never will IRL). There's a line further down that sets `hasMobileNumber` to `true` if `null` were specified and `UserType == Default`.

Reply:
Ah missed that!